package.json

```diff
@@ -2,12 +2,13 @@
   "name": "libxmljs",
   "author": "Marco Rogers",
   "contributors": [
-    "Jeff Smick"
+    "Jeff Smick",
+    "Martin von Gagern <Martin.vGagern@gmx.net>"
```

Comment:
Im glad you have a strong opinion on this. Like I said, this field **is not actively used by us** and therefore I don't feel that updating it is required. Additionally, do you see my name? Or the name of any other contributor? And furthermore. The changes have not been merged in or contributions added. I do not appreciate when folks automatically add themselves to such lists (when they are maintained). These lists are for project collaborators to decide who is added. The new maintainers are free to revive this list and determine rules for keeping it up to date but if you need your name listed in a file to allow your contributions to be accepted, then maybe you are contributing to a project for the wrong reasons.